org.civicrm.reqangular/src/common/templates/date-picker.html

```diff
@@ -0,0 +1,42 @@
+<div class="input-group chr_uib-date-picker">
```

Comment:
where is the logic to display one date picker or the other?

Reply:
Its handled from the CSS, 
https://github.com/compucorp/org.civicrm.shoreditch/pull/41/files#diff-b564bbd3a051a440df22b7b52d761b88R4 and
https://github.com/compucorp/org.civicrm.shoreditch/pull/41/files#diff-b564bbd3a051a440df22b7b52d761b88R38